campaigns/signup-snippet/snippet.html

```diff
@@ -0,0 +1,1458 @@
+<div class="snippet" id="signup-snippet-{{ snippet_id }}">
+  <style type="text/css">
+    @font-face {
+      font-family: 'fontello-{{ snippet_id }}';
```

Comment:
TIP:  Uniquely identifying elements using `{{ snippet_id }} ` is not required anymore.